doc/api/typescript.md

```diff
@@ -44,13 +53,15 @@ added: v22.6.0
 > Stability: 1.0 - Early development
 
 The flag [`--experimental-strip-types`][] enables Node.js to run TypeScript
-files that contain only type annotations. Such files contain no TypeScript
-features that require transformation, such as enums or namespaces. Node.js will
-replace inline type annotations with whitespace, and no type checking is
-performed. TypeScript features that depend on settings within `tsconfig.json`,
+files. By default Node.js will execute only files that contain no
+TypeScript features that require transformation, such as enums or namespaces.
+Node.js will replace inline type annotations with whitespace,
+and no type checking is performed.
+To enable the transformation of such features
+use the flag [`--experimental-transform-types`][].
+TypeScript features that depend on settings within `tsconfig.json`,
 such as paths or converting newer JavaScript syntax to older standards, are
-intentionally unsupported. To get fuller TypeScript support, including support
-for enums and namespaces and paths, see [Full TypeScript support][].
+intentionally unsupported. To get fuller TypeScript support, see [Full TypeScript support][].
```

Comment:
FWIW "Fuller", AFAIK, refers to a cloth worker / blacksmith, and not "more full". Could a different word be used instead?

https://www.merriam-webster.com/dictionary/fuller